demos/serverless/template.yaml

```diff
@@ -32,8 +32,14 @@ Parameters:
     Description: Prefix of S3 bucket to write capture artifacts.  The bucket name will be suffixed with the region.
     Default: ""
     Type: String
+  UseFetchCredentialLambda:
+    Description: Set to true to deploy a lambda to return AWS credentials. This should be disabled in most demo.
```

Comment:
Can you please update this description to specify which AWS credentials? I will prefer it to be something like:
`Set to true to deploy a lambda to return AWS credentials with limited privileges`.

Reply:
Updated.